src/Plugin/dgi_migrate_alter/foxml/DgisNodesAlter.php

```diff
@@ -0,0 +1,287 @@
+      'plugin' => 'single_value',
+    ];
+    $process['field_related_item_paragraph'][3]['values']['field_related_item_genre'][] = [
+      'plugin' => 'callback',
+      'callable' => 'array_filter',
+    ];
+    $process['field_related_item_paragraph'][3]['values']['field_related_item_genre'][] = [
+      'plugin' => 'null_coalesce',
+    ];
+
+    $process['_rights_statement'] = $process['_resource_type'];
+    $process['_rights_statement'][0]['query'] = 'mods:accessCondition[@type="rights statement"]';
+    $process['_rights_statement'][4]['values']['_vid'][0]['default_value'] = 'rights_statements';
+
+    $process['_unspecified_rights_statement'] = $process['_unspecified_resource_type'];
+    $process['_unspecified_rights_statement'][1]['use_as_key'] = '@_rights_statement';
+    $process['_unspecified_rights_statement'][2]['values']['_vid'][0]['default_value'] = 'rights_statements';
+
+    $process['field_rights_statement'] = $process['field_resource_type'];
+    $process['field_rights_statement'][0]['source'] = [
+      '@_rights_statement',
+      '@_unspecified_rights_statement',
+    ];
+
+    $process['field_publication_genre'][3] = [
+      'plugin' => 'single_value',
+    ];
+
+    array_splice($process['field_publication_genre'], 4, 0, [
+      [
+        'plugin' => 'callback',
+        'callable' => 'array_filter',
+      ],
+      [
+        'plugin' => 'null_coalesce',
+      ],
+    ]);
+
+    $process['field_publication_title'][3] = [
+      'plugin' => 'single_value',
+    ];
+
+    array_splice($process['field_publication_title'], 4, 0, [
+      [
+        'plugin' => 'callback',
+        'callable' => 'array_filter',
+      ],
+      [
+        'plugin' => 'null_coalesce',
+      ],
+    ]);
+
+    $process['field_remote_media_url'] = $process['field_ismn'];
+    $process['field_remote_media_url'][0]['query'] = 'mods:identifier[@displayLabel="remote media URL"]';
+
+    $to_remove = [
+      ['field_version_identifier'],
+      ['field_resource_publication_statu'],
+      ['field_change_note'],
+      ['field_purl'],
+      ['field_handle'],
+      ['field_open_url'],
+      ['field_ismn'],
+      ['field_repec'],
+      ['field_gpo_number'],
+      ['field_oclc_number'],
+      ['field_pubmed_number'],
+      ['field_faceted_subject'],
+      ['field_sudoc_number'],
+      ['field_swank_classification'],
+      ['field_state_gov_classification'],
+      ['field_publication_volume_title'],
+      ['field_publication_chapter_number'],
+      ['field_publication_section'],
+      ['field_publication_url'],
+      ['field_origin_information', 3, 'values', '_field_date_created_single'],
+      ['field_origin_information', 3, 'values', '_field_date_created_start'],
+      ['field_origin_information', 3, 'values', '_field_date_created_end'],
+      ['field_origin_information', 3, 'values', '_field_date_created_assembled'],
+      ['field_origin_information', 3, 'values', 'field_date_created'],
```

Comment:
Probably should nuke the backup field as well:

```
                'field_date_created_backup' => array (1) [
                    0 => array (5) [
                        'plugin' => string (4) "gate"
                        'source' => string (30) "@_field_date_created_assembled"
                        'use_as_key' => string (19) "@field_date_created"
                        'valid_keys' => array (1) [
                            0 => null
                        ]
                        'key_direction' => string (6) "unlock"
                    ]
                ]
```

Reply:
Removed.